MSAL/src/public/configuration/MSALPublicClientApplicationConfig.h

```diff
@@ -78,6 +78,14 @@ to target MSAL at a specific test slice & flight. These apply to all requests ma
 /** MSAL configuration interface responsible for token caching and keychain configuration. Refer to `MSALCacheConfig` for more details */
 @property (readonly) MSALCacheConfig *cacheConfig;
 
+/**
+ For clients that support multiple national clouds, set this to YES. NO by default.
+ If set to YES, the Microsoft identity platform will automatically redirect user to the correct national cloud during the authorization flow. You can determine the national cloud of the signed-in account by examining the authority associated with the MSALResult. Note that the MSALResult doesn't provide the national cloud-specific endpoint address of the resource for which you request a token.
+ 
+ @note Your client_id needs to be registered in national clouds for this feature to work.
+ */
+@property (nonatomic) BOOL multipleCloudsSupported;
```

Comment:
Thanks for not using instanceAware in public api. This name is much better :)

Reply:
Name is courtesy of the Android team :)